doc/script_commands.txt

```diff
@@ -9475,6 +9475,23 @@ Examples:
 
 ---------------------------------------
 
+*instance_list(<"map name">{,<instance mode>});
+
+Set '.@instance_list' as an array for the instances ids with the map name and the mode passed.
+Return '.@instance_list' array size.
+
+instance mode IM_NONE, IM_CHAR, IM_PARTY, IM_GUILD, IM_CLAN
+if the instance mode is not provided By default it will return all the instances ids for that map
```

Comment:
```suggestion
Instance mode options: IM_NONE, IM_CHAR, IM_PARTY, IM_GUILD, or IM_CLAN
If the instance mode is not provided then it will return all the instance IDs for that map.
```